src/main/java/com/github/dockerjava/api/model/PullResponseItem.java

```diff
@@ -22,8 +26,9 @@ public boolean isPullSuccessIndicated() {
             return false;
         }
 
-        return (getStatus().contains("Download complete") || getStatus().contains("Image is up to date") || getStatus()
-                .contains("Downloaded newer image"));
+        return (getStatus().contains(DOWNLOAD_COMPLETE) || getStatus().contains(IMAGE_UP_TO_DATE)
```

Comment:
Not sure I understand. What were you envisioning? Using explicit if statements? Not sure if that is clearer.

Besides, I am trying to minimize the size of this commit to just fix the issue identified. I am happy to help with code cleanup and formatting in another issue/PR if that is useful but that seems outside the scope of this change.